package.json

```diff
@@ -33,10 +33,12 @@
     "mkdirp": "^1.0.4",
     "nps": "^5.10.0",
     "prettier": "^2.4.1",
+    "prop-types": "^15.7.2",
     "puppeteer": "^16.0.0",
     "raw-loader": "^4.0.0",
     "rimraf": "^3.0.0",
     "serve": "^14.0.1",
+    "spectacle-mdx-loader": "workspace:*",
```

Comment:
These deps should be in `examples/mdx/package.json`

Reply:
these have been moved 👍